Anlab.Mvc/Client/components/OrderForm.tsx

```diff
@@ -74,7 +74,7 @@ export default class OrderForm extends React.Component<undefined, IOrderState> {
 
             initialState.quantity = orderInfo.Quantity;
             initialState.additionalInfo = orderInfo.AdditionalInfo;
-            initialState.additionalEmails = orderInfo.AdditionalEmails;
+            initialState.additionalEmails = orderInfo.AdditionalEmails != null? orderInfo.AdditionalEmails : [];

```

Comment:
@jSylvestre try to never use double equals in javascript, but also what I would do here is just guarantee on the server-side that you never get back a null array for additional emails, in the ctor() have it new up an empty array